migrations/schema-15.sql

```diff
@@ -1003,3 +1003,45 @@ CREATE EVENT TRIGGER pgrst_drop_watch ON sql_drop
 -- Dbmate schema migrations
 --
 
+INSERT INTO public.schema_migrations (version) VALUES
+    ('00000000000000'),
+    ('00000000000001'),
+    ('00000000000002'),
+    ('00000000000003'),
+    ('10000000000000'),
+    ('20211115181400'),
+    ('20211118015519'),
+    ('20211122051245'),
+    ('20211124212715'),
+    ('20211130151719'),
+    ('20220118070449'),
+    ('20220126121436'),
```

Comment:
@samrose Q: Where do this migrations rows come from? And why are they necessary?

Reply:
Also why so many rows from a simple change?

Reply:
@steve-chavez this PR is marked as a draft and so is not ready for review

you may recall this question being discussed here yesterday and answered 

https://github.com/supabase/postgres/pull/1555#discussion_r2049738715

These files are strictly automated output, and should never be edited or changed. It's common to see duplication and large amounts of output from automated file generation.

They are automated output from the https://github.com/amacneil/dbmate migration tool, which is the current tool in use for managing and updating database migrations in `supabase/postgres` 

Those particular rows are the "id" of the migrations applied as tracked by the dbmate tool

https://github.com/amacneil/dbmate?tab=readme-ov-file#schema-file

Reply:
Here is where you pointed that this was answered yesterday https://github.com/supabase/postgres/pull/1554#discussion_r2049908882

Reply:
I recommend looking at the documentation for dbmate linked in the repo to understand more about this tool, why it produces these files and how we use it etc https://github.com/amacneil/dbmate?tab=readme-ov-file#schema-file

Reply:
@samrose Sorry, although to be fair yesterday I asked about the generated files and not the `schema_migrations` rows.

I'll read the documentation of that tool.

---

That being said, there's always cognitive load from these tools. This is one of the reasons I was proposing to rely on a native pg extension instead, then we can forget about these details (if we manage to pull it through).

Reply:
@steve-chavez you are right your question was different thanks for pointing that out. 

I am all in on your RFC in the meantime we have to use this approach until we can change it. Thank you for reviewing this too of course